lib/libzfs/libzfs_pool.c

```diff
@@ -3852,8 +3853,13 @@ zpool_history_unpack(char *buf, uint64_t bytes_read, uint64_t *leftover,
 		/* add record to nvlist array */
 		(*numrecords)++;
 		if (ISP2(*numrecords + 1)) {
-			*records = realloc(*records,
+			tmp = realloc(*records,
 			    *numrecords * 2 * sizeof (nvlist_t *));
+				if (tmp == NULL) {
```

Comment:
Github shows 1 tab as twice wider, visually it's the same as `*numrecords...` line in any text editor.

I'll fix it if you say so, thanks.